backend/apid/routers/graphql.go

```diff
@@ -28,9 +30,22 @@ type GraphQLRouter struct {
 }
 
 // NewGraphQLRouter instantiates new events controller
-func NewGraphQLRouter(apiURL string, tls *tls.Config, store store.Store) *GraphQLRouter {
-	factory := restclient.NewClientFactory(apiURL, tls)
-	service, err := graphql.NewService(graphql.ServiceConfig{ClientFactory: factory})
+func NewGraphQLRouter(store store.Store, eventStore store.EventStore, auth authorization.Authorizer, qGetter types.QueueGetter, bus messaging.MessageBus) *GraphQLRouter {
+	service, err := graphql.NewService(graphql.ServiceConfig{
+		AssetClient:       api.NewAssetClient(store, auth),
+		CheckClient:       api.NewCheckClient(store, actions.NewCheckController(store, qGetter), auth),
+		EntityClient:      api.NewEntityClient(store, eventStore, auth),
+		EventClient:       api.NewEventClient(eventStore, auth, bus),
+		EventFilterClient: api.NewEventFilterClient(store, auth),
+		HandlerClient:     api.NewHandlerClient(store, auth),
+		MutatorClient:     api.NewMutatorClient(store, auth),
+		SilencedClient:    api.NewSilencedClient(store, auth),
+		NamespaceClient:   api.NewNamespaceClient(store, auth),
+		HookClient:        api.NewHookConfigClient(store, auth),
+		UserClient:        api.NewUserClient(store, auth),
+		RBACClient:        api.NewRBACClient(store, auth),
+		GenericClient:     &api.GenericClient{Store: store, Auth: auth},
+	})
 	if err != nil {
 		logger.WithError(err).Panic("unable to configure graphql service")
 	}
```

Comment:
Arguably out of scope of this PR but lines 70 through 77 should no longer relevant.

https://github.com/sensu/sensu-go/blob/5a8ed856967ef3ee4dc0ffdef2f27eadee0a2d8c/backend/apid/routers/graphql.go#L70-L77

Reply:
OK, I'll nuke these lines.